source/system/structs/List.ooc

```diff
@@ -16,6 +16,8 @@ List: abstract class <T> {
 	remove: abstract func ~last -> T
 	remove: abstract func ~atIndex (index: Int) -> T
 	removeAt: abstract func (index: Int)
+	removeAt: abstract func ~range (range: Range)
```

Comment:
This could have been
```ooc
removeAt: func ~range (range: Range) { this removeAt(range min, range max)) }
removeAt: abstract func ~indices (start, end: Int)
```
that way you don't have to implement it everywhere.